tech/languages/python/multiple-pythons.md

```diff
@@ -49,20 +50,19 @@ Note that most of the CPython versions, other than those provided by the
 `python2` and `python3` packages, are only provided for developer's convenience
 so you can run your test suites. Critical security bugs might not be always
 fixed and those interprets are not intended for production.
+Especially the development version might be quite unstable.
 
 ## Getting it and running it all with tox
 
 [Tox](https://tox.readthedocs.io/) is tool that helps you test your Python code
 on multiple Pythons. If you install it on Fedora via the dnf package manager,
-you'll automatically get all the CPythons and PyPys:
+you'll automatically get all the CPythons (except the development version)
+and PyPys:
```

Comment:
Note that there's no need to wrap lines here, as `jekyll` does that during web pages generation...
I think that it's a bit unnecessary (this is 80 chars, right? - I'd prefer at least 120).

Reply:
This wasn't wrapped for Jekyll, but for whoever reads/edits the source. It more or less matches the length of the surrounding lines.